lib/backend/syncersv1/updateprocessors/workloadendpointprocessor.go

```diff
@@ -114,17 +114,26 @@ func convertWorkloadEndpointV2ToV1Value(val interface{}) (interface{}, error) {
 		})
 	}
 
+	// Add the namespace and orcehstrator labels to the set of labels we give Felix.  We
+	// don't need to persist these, and we need to make sure they are not user-configurable
+	// so we set them here and we also return them in the WEP resource in the client.
+	labels := v2res.GetLabels()
+	if labels == nil {
+		labels = make(map[string]string, 2)
+	}
+	labels[apiv2.LabelNamespace] = v2res.Namespace
+	labels[apiv2.LabelOrchestrator] = v2res.Spec.Orchestrator
+
 	v1value := &model.WorkloadEndpoint{
 		State:            "active",
 		Name:             v2res.Spec.InterfaceName,
-		ActiveInstanceID: v2res.Spec.ContainerID,
```

Comment:
Why this change?

Reply:
It's not required for Felix.

Reply:
Yep. I was thinking along the lines of "what if another syncer client comes along that wants it" - e.g. I'd like to eventually update the kube-controllers code to use a syncer rather than a periodic poll, and it does care about this field.

I think I wouldn't use an updateprocessor to convert to v1 in that case though, so it's probably fine.